src/main/java/creativei/vo/AddressVo.java

```diff
@@ -18,6 +18,17 @@
 
     public AddressVo(){}
 
+    public AddressVo(InquiryAddress inquiryAddress){
+        this.pin=inquiryAddress.getPincode();
+        this.addressLine1=inquiryAddress.getAddressLine1();
+        this.id=inquiryAddress.getId();
+        this.locationId=inquiryAddress.getLocation().getId();
+        this.city=inquiryAddress.getLocation().getCity().getName();
```

Comment:
Add null check for all values dependent on location. If location is null, it will throw exception

Reply:
Location is required field if location is not set already an exception INVALID PARAM REQUEST will thrown.

Reply:
Ok. Still as we are making a constructor it would be better to be proactive and handle null condition to avoid unchecked exceptions

Reply:
issues resolved in commit 4025cc9